README.rst

```diff
@@ -17,3 +17,37 @@ and development purposes you can use `message_factory(...)` which allows you to
     message_factory(t_name=<str>, message_contents=<Tuple[str, str]>, [num_partitions=<int>], [replication_factor=<int>])
 
 bracketed arguments are optional.
+
+Development and Testing
+#######################
+For development and testing it is very useful to have a local instance of Kafka running on your machine
+
+We provide a docker-compose file to get you up and running quickly with a basic Kafka server; this file is based on
+`Single Zookeeper/Multiple Kafka<https://github.com/simplesteph/kafka-stack-docker-compose#single-zookeeper--multiple-kafka`.
+
+In order to start Zookeeper as well as the Kafka Servers simply run `$ podman-compose up` or `$ docker-compose up`,
+choose the appropriate option based on the system which you are using.
+
+Once you have Kafka up and running you should be ready to begin coding your own messaging producers and consumers.  The
+interface between `Kafka` and `Python` is handled by a library called `Faust<https://faust.readthedocs.io/en/latest/>_`.
+Faust's documentation will be extremely helpful to you when you are developing your own applications. If you would like
+examples of producers and consumers from Team Thoth, look at the following two repositories, `investigator
+<https://github.com/thoth-station/investigator>_` and `package-update<https://github.com/thoth-station/package-update-job>_`.
+
+You may find it useful to use console producers and consumers while testing your, to create one simply attach a bash shell
+to one of your Kafka Servers by running: `$ podman exec -it messaging_kafka1_1 bash`, your container names should be
+the same as given here, if not, run `$ podman ps` and choose the correct container.  These containers have all Kafka
+binaries in appropriate places so you can simply run `$ kafka-console-consumer`, `$ kafka-console-producer`, or any other
+kafka command that you may find useful.
+
+example:
+```
+$ bin/kafka-console-consumer.sh --bootstrap-server localhost:9092 --topic test --from-beginning
```

Comment:
Inside the containers bash this seems to work - `kafka-console-consumer --bootstrap-server localhost:9092 --topic test --from-beginning`

Reply:
Yes, sorry. No `bin/` is required. I'll change this